config/esp32/components/chip/Kconfig

```diff
@@ -130,6 +130,13 @@ menu "CHIP Core"
             help
                 Some device types don't require the read client. Enabling this option may save some flash/ram.
 
+        config ENABLE_PERSIST_SUBSCRIPTIONS
+            bool "Enable persist subscriptions"
+            default n
```

Comment:
@wqx6 Is there a specific reason that the persistent subscriptions are disabled by default?

Reply:
To avoid the issue in #30491, Also in our test, the HomePod mini could re-establish the subscriptions quickly when rebooting the devices for both the Wi-Fi and Thread modules. So disable the persistent subscriptions by default.